python/covid_web/covid_web/rest.py

```diff
@@ -3,24 +3,26 @@
 
```

Comment:
There will be a lot of conflicts in this file with https://github.com/lukipuki/COVID-19-simulation/pull/101, we should resolve the conflicts carefullly.

Reply:
I'm closing #101 for now and I'll handle them once this gets merged. Meanwhile, you may review updated version https://github.com/lukipuki/COVID-19-simulation/pull/103